uvicorn/main.py

```diff
@@ -319,6 +319,13 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     help="Ciphers to use (see stdlib ssl module's)",
     show_default=True,
 )
+@click.option(
+    "--ssl-context-factory",
+    type=typing.Callable,
```

Comment:
This can't be a callable. What I said was to load the function object from the path... Which I'm not sure if we really should make this available via CLI.

Reply:
There are messages I've written here that you ignored. I'm not reviewing this again without them addressed.

The code changes here don't fully take in consideration my previous comments.